### PR TITLE
 implementing awesome-books with ES6

### PR DESCRIPTION
**Pull request by ANTOINE1128**
in this pull request I have made the following changes :
implementing the use of ES6 
changing functions into arrow functions
setting a date 
creating modules

### CODE REVIEW

File: display.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  document.querySelector('#book-list').style.display = 'none';
+  document.querySelector('#book-form').style.display = 'block';
+  document.querySelector('#contact').style.display = 'none';
+}
+
+export function displayContact() {
+  document.querySelector('#book-list').style.display = 'none';
+  document.querySelector('#book-form').style.display = 'none';
+  document.querySelector('#contact').style.display = 'block';
+}
```
Comment (V-Blaze) [inline display.js]: -  [x] Good job so far, :+1: :+1: I think there are some improvements you need to make on these functions according to the project requirements.
> Refactor your functions to arrow functions.

- - According to the project requirement, you are meant to refactor all your functions to use the ES6 arrow function, I'll suggest you follow the requirement and refactor these functions. :+1: :smile:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        </div>
+      </div>
+    </section>
+
+    
+    <script src="./storage.js" type="module"></script>
+    <script src="./UI.js" type="module"></script>
+    <script src="./display.js" type="module"></script>
+    <script src="./index.js" type="module"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/luxon/2.0.0/luxon.min.js"></script>
```
Comment (V-Blaze) [inline index.html]: - [x] Following the project requirement, you are meant to only import the `index.js`  and `index.css` file here.
> Create the main index.html file. Link your index.js and index.css files there.
- [x] Other js files should be imported in the `index.js` file. :+1: :+1: . I'll suggest you visit your codebase and implement this. :+1:

File: index.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  const currentDate = document.getElementById('current-date');
+  const date = DateTime.now().toLocaleString();
+  if (currentDate) {
+    currentDate.textContent = date;
+  }
+});
+
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  // Remove book from UI
+  UI.deleteBook(e.target);
```
Comment (V-Blaze) [inline index.js]: -  [x] I'll suggest you move these event listeners into a function declaration and then call them when needed, :+1: That will make your codebase clean and more easy to maintain. :smile: :wave:

File: index.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { DateTime } from 'luxon';
+import { Store } from './storage.js';
+import { UI } from './UI.js';
+import { displayList, displayForm, displayContact } from './display.js';
```
Comment (V-Blaze) [inline index.js]: - [x] Great job installing the `Luxon` library, :+1:  but currently, the current data is not visible in the UI, I'll suggest you check your implementation and fix this as its part of the requirement for this milestone. :+1: :wave: 

- [x] You did great breaking down your functions into files, But I think your code will be more organized and professional if you create a `module` folder in your application root and then move all the js files into the folder. :+1: :smile:  

- [x] As part of the requirement, After refactoring your code, The functionality of adding and removing book should still work, but that's not the case here. I think you should go through the code one more time to get these features to be functional. :smile: :wave: :100: 

- - VIDEO FOR REFERENCE:

https://user-images.githubusercontent.com/107632780/236909019-d23828d3-fb8c-42f2-83af-45e8384b1672.mp4

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @ANTOINE1128 , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Github flow✔
- LInters are green👋 
-  Installed Luxon library package✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**